cves/kernel/CVE-2019-19045.yml

```diff
@@ -387,38 +413,39 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: This vulnerability shows the importance of having multiple layers of security. Even if one function fails, others should be in place to prevent exploitation.
   least_privilege:
-    applies:
-    note:
+    applies: false
+    note: The vulnerability does not appear to be related to the principle of least privilege but rather to improper memory handling.
```

Comment:
Repeated information here about the vulnerability being related to improper memory handling

Reply:
Thank you for pointing that out!